low_level/comms.py

```diff
@@ -47,7 +80,35 @@ class CommsHandler:
     def __init__(self, port, baud_rate):
         """ Initialise the rx_listener and serial. """
         self.rx_state_machine = StateMachine()
-        self.comms = SerialComms(port, baud_rate)
+        if RaspberryPi is True:
+            self.radio = RadioComms(spi, CS, RESET, 434.0)
+        self.serial = SerialComms(config.COM_PORT, config.BAUD_RATE)
+
+    def change_comms(self):
+        if config.COMMS == "RF69":
+            return self.radio
+        elif config.COMMS == "SERIAL":
```

Comment:
What happens if you select "SERIAL" on the RaspPi? Does it crash nastily?

Reply:
Ok, can you show me it running on both the Pi and Windows so I can try and kill it?

Reply:
Demoed, all ok... Can now be resolved